Description: arm targets armv7 by default, try to fix this.
Forwarded: not-needed, upstream do not promote armv4t support
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2011-12-01
--- a/Makefile
+++ b/Makefile
@@ -163,7 +163,7 @@
 
 $(OUTDIR)/Makefile-arm: $(GYPFILES) $(ENVFILE)
 	gyp --generator-output="$(OUTDIR)" build/all.gyp \
-	              -Ibuild/standalone.gypi --depth=. -Ibuild/armu.gypi \
+	              -Ibuild/standalone.gypi --depth=. -Dtarget_arch=arm \
 	              -S-arm $(GYPFLAGS)
 
 # Replaces the old with the new environment file if they're different, which
